res/layout/files.xml

```diff
@@ -20,39 +20,21 @@
 <android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/drawer_layout"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:clickable="true" >
+    android:layout_height="match_parent">
 
-    <!-- The main content view -->
     <LinearLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="@color/background_color"
-        android:baselineAligned="false"
-        android:orientation="horizontal"
-        android:id="@+id/ListLayout"
-        android:contentDescription="@string/list_layout"
+        android:orientation="vertical"
         >
 
+        <!-- toolbar -->
+        <include layout="@layout/toolbar" />

```

Comment:
Yes, a proper drawer menu should be done via a toolbar not an action bar. In order to use a toolbar one musst move to a theme without an action since else you basically have two bars which crashes within android implementation.

The reason for the toolbar is that is a more flexible widget for actionsbars, see http://android-developers.blogspot.kr/2014/10/appcompat-v21-material-design-for-pre.html - If you take a look at http://www.google.com/design/spec/layout/structure.html#structure-toolbars this is basically Toolbar but would most of the time not be possible with an action bar.